### ISSUE TITLE
Formatter flattens JavaScript

### ISSUE THREAD

**Issue by arnaldop** [karatelabs/intellij-plugin#54]
This:
```js
    * def sleep =
      """
        function(pause) {
          console.log("Sleeping for " + pause + " milliseconds.");
          java.lang.Thread.sleep(pause);
          console.log("Sleeping for " + pause + " milliseconds.");
        }
      """
```
becomes this:
```js
    * def sleep =
      """
      function(pause) {
      console.log("Sleeping for " + pause + " milliseconds.");
      java.lang.Thread.sleep(pause);
      console.log("Sleeping for " + pause + " milliseconds.");
      }
      """
```

**Comment by arnaldop** [comment]
@ptrthomas, you can see in #55 an example of how it looks with a longer function. And I have some even longer ones!

**Comment by ptrthomas** [comment]
@arnaldop this will be fixed in the next release for sure

**Comment by ptrthomas** [comment]
while 2.0.0 has been finally released to the marketplace, the fix for this ticket is available here: https://github.com/karatelabs/intellij-plugin/releases/tag/v2.0.0.EAP (2.0.1.31)

**Comment by arnaldop** [comment]
@ptrthomas, I added all of the missing semi-colons but it still flattens. Note `createAgent`. I included the beginning of the file to show that nothing else is messing with the formatter before that snippet. Everything is kosher. It is all valid Karate and the Plugin does not warn me of any issues.

```
Feature: bulk

  Background:

    * callonce read('classpath:utils.feature')
    * configure retry = { count: 10, interval: 5000 }

    * def originalLogLevel = getLogLevel()
    #    * setLogLevel('ERROR')

    Given url baseAppUrl

    * call read('this:../user/register.reusable.feature')
    * call read('this:../user/login.reusable.feature')

    * def createAgent =
      """
      function(count) {
        var ids = [];
  
        for (var i = 0; i < count; i++) {
          var testData = {
            agent: {
              lastName: 'Last',
              firstName: 'A' + i
            }
          };
          karate.set('testData', testData);
    
          karate.call('../agent/create-agent.reusable.feature');
    
          ids.push(karate.get('testData').agent.id);
        }
  
        karate.get('ids').agents = ids;
      }
      """
```

After
```
Feature: bulk

  Background:

    * callonce read('classpath:utils.feature')
    * configure retry = { count: 10, interval: 5000 }

    * def originalLogLevel = getLogLevel()
    #    * setLogLevel('ERROR')

    Given url baseAppUrl

    * call read('this:../user/register.reusable.feature')
    * call read('this:../user/login.reusable.feature')

    * def createAgent =
      """
      function(count) {
      var ids = [];

      for (var i = 0; i < count; i++) {
      var testData = {
        agent: {
          lastName: 'Last',
          firstName: 'A' + i
        }
      };
      karate.set('testData', testData);

      karate.call('../agent/create-agent.reusable.feature');

      ids.push(karate.get('testData').agent.id);
      }

      karate.get('ids').agents = ids;
      }
      """
```

**Comment by ptrthomas** [comment]
@arnaldop thanks - will reopen to fix for-loop blocks

**Comment by ptrthomas** [comment]
should be fixed in 2.0.1 in the marketplace https://plugins.jetbrains.com/plugin/19232-karate/edit/versions/stable/493388
